README.md

```diff
@@ -1,3 +1,29 @@
 # fabric8-gemini-server
 Fabric8-server analytics powered services to initiate and report the readiness of
 all registered services
+
+### ENDPOINTS
+
+#### POST: /api/v1/register
+```
+{
+	"git_url" : <some_url>,
+	"git_sha" : <some_sha>,
+	"email_ids" : <some_email>
+}
+
+```
+
+##### Result :
+```
+{
+  "data": {
+    "email_ids": <some_email>,
+    "git_sha": <some_sha>,
+    "git_url": <some_url>,
+    "last_scanned_at": "Mon, 05 Mar 2018 00:00:00 GMT"
```

Comment:
I did not understand why should a response to `POST` call return the data itself? The response to `POST` could just contain the acknowledge receipt related data. For example, as other fields in response has, `success` and `summary`.

Reply:
@navidshaikh, in my understanding, this was more detailed explanation. So I added this detail in response.

Reply:
I dont think this a right PR to discuss about what's returned as the response to POST call. However thinking, from client side point of view, client has already invoked POST call with same data, all it needs is, acknowledgment. Client is not going to use that data, taking that data off should simplify the response, processing bits and have lightweight API calls.

Reply:
@navidshaikh, I am open to discussion regarding this. Just the fact that I am using the return data in tests.